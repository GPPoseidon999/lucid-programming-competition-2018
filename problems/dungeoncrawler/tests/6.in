14
depth breadth
da deptha 0
db depthb 1 deptha
dc depthc 1 depthb
dd depthd 1 depthc
dfinal depth 1 depthd
ba breadtha 0
bb breadthb 0
bc breadthc 0
bd breadthd 0
be breadthe 0
bf breadthf 0
bg breadthg 0
bh breadthh 0
bfinal breadth 7 breadtha breadthb breadthc breadthd breadthe breadthf breadthg