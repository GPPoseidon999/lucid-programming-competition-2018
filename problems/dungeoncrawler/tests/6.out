da db dc dd dfinal ba bb bc bd be bf bg bfinal